test/aria-practices/apg.spec.js

```diff
@@ -0,0 +1,88 @@
+const path = require('path');
+const fs = require('fs');
+const { createServer } = require('http');
+const express = require('express');
+const chromedriver = require('chromedriver');
+const AxeBuilder = require('@axe-core/webdriverjs');
+const testListen = require('test-listen');
+const { getWebdriver, connectToChromeDriver } = require('./run-server');
+const { assert } = require('chai');
+const globby = require('globby');
+
+describe('aria-practices', function () {
+  // Use path.resolve rather than require.resolve because APG has no package.json
+  const apgPath = path.resolve(__dirname, '../../node_modules/aria-practices/');
+  const filePaths = globby.sync(`${apgPath}/examples/**/*.html`)
+  const festFiles = filePaths.map(fileName => fileName.split('/aria-practices/examples/')[1])
+  const port = 9515;
+  let driver, server, addr, axeSource;
+  this.timeout(50000);
+  this.retries(3);
+
+  before(async () => {
+    const axePath = require.resolve('../../axe.js');
+    axeSource = fs.readFileSync(axePath, 'utf8');
+    chromedriver.start([`--port=${port}`]);
+    await new Promise(r => setTimeout(r, 500));
+    await connectToChromeDriver(port);
+
+    const app = express();
```

Comment:
Are any of the test files using absolute paths? If not I think our `npm run start` command should server node_modules directly as well, so we might not need express for this.